docs/attributes-registry/db.md

```diff
@@ -166,6 +166,16 @@
 | `ExecuteJavaScript` | execute_javascript |
 <!-- endsemconv -->
 
+<!-- semconv registry.db(omit_requirement_level,tag=tech-specific-cosmosdb) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| [`rntbd.url`](../attributes-registry/db.md) | string | CosmosDB server url contacted | `rntbd://some-url/` |
+| [`rntbd.status_code`](../attributes-registry/db.md) | string | Status Code returned by cosmos DB server | `200` `404` etc. |
+| [`rntbd.sub_status_code`](../attributes-registry/db.md) | string | Sub Status Code which tells more about the failure | `20001` `20002` etc |
+| [`rntbd.*`](../attributes-registry/db.md) | string | Cosmos DB specific information, it can vary with different flavours of SDK |  |
```

Comment:
do you want to reserve a namespace? If so,  you're reserving it by declaring any attribute in this namespace.

There is no point in declaring an open set of attributes if you don't document each of them

Reply:
makes sense actually, I will remove them